src/Navbar.js

```diff
@@ -210,13 +209,13 @@ class Navbar extends React.Component {
 }
 
 Navbar.propTypes = propTypes;
-Navbar.defaultProps = defaultProps;
 
 const DecoratedNavbar = createBootstrapComponent(
   uncontrollable(Navbar, { expanded: 'onToggle' }),
   'navbar',
 );
 
+DecoratedNavbar.defaultProps = defaultProps;
```

Comment:
why this change?

Reply:
In order to pass the "Should define default "as" in prop destructuring" tests. If those tests aren't necessary then this doesn't have to be there.